c/common/adapters/lock_win32.c

```diff
@@ -6,6 +6,12 @@
 #include "malloc.h"
 #include "iot_logging.h"
 
+#ifdef WINCE
+
+const char* Lock_ResultAsString[] = { "OK", "ERROR" };
```

Comment:
The macro used to translate the enum into a string array is not "digested" by VS2013 compiler. I will investigate a bit more, but this seemed to be the only workaround.